### PR TITLE
Fix image path

### PR DESCRIPTION
**Pull request by catong**
Fixes image paths to use lowercase "i" in shared/images folder paths. Changes made in source folders that refer to shared/images. 
@lamagnifica sanity check please? I checked builds and random files for correct display of images.

### CODE REVIEW

File: en_us/olx/source/problem-xml/create_problem.rst [hunk 1]
```diff
@@ -32,9 +32,9 @@ The Learner View of a Problem
 
 All problems on the edX platform have several component parts.
 
-.. image:: ../../../shared/Images/AnatomyOfExercise1.png
- :alt: Image of a problem from a learner's point of view, with callouts for
-       elements of the problem.
+.. image:: ../../../shared/images/AnatomyOfExercise1.png
+ :alt: Image of a problem from a student's point of view, with callouts for
+       elements of the problem
```
Comment (lamagnifica) [inline en_us/olx/source/problem-xml/create_problem.rst]: (nit) the alt seemed better as it was...

File: en_us/olx/source/problem-xml/create_problem.rst [hunk 2]
```diff
@@ -68,15 +68,15 @@ All problems on the edX platform have several component parts.
    for individual problems defaults to that number, and cannot be set to
    unlimited.
 
-   .. image:: ../../../shared/Images/AnatomyOfExercise2.png
-    :alt: Image of a problem from a learner's point of view, with callouts for
-          attempts and showing the answer.
+   .. image:: ../../../shared/images/AnatomyOfExercise2.png
+    :alt: Image of a problem from a student's point of view, with callouts for
+          attempts and showing the answer
```
Comment (lamagnifica) [inline en_us/olx/source/problem-xml/create_problem.rst:73]: alt

Comment (catong) [inline en_us/olx/source/problem-xml/create_problem.rst:73]: Hm, those were merge conflicts that I thought I resolved correctly with "learner". I'll fix.

File: en_us/olx/source/problem-xml/create_problem.rst [hunk 3]
```diff
@@ -94,13 +94,17 @@ All problems on the edX platform have several component parts.
...
+   .. image:: ../../../shared/images/AnatomyOfExercise3.png
     :alt: Image of a problem in the left pane
 
+
 #. **Grading.** You can specify whether a group of problems is graded. If a
    group of problems is graded, an icon of a pen and a piece of paper appears
    for that assignment in the left pane.
 
+   .. image:: ../../../shared/images/clock_icon.png
+
```
Comment (lamagnifica) [inline en_us/olx/source/problem-xml/create_problem.rst]: must have an alt (not a nit), and the clock icon isn't used any more, it was replaced by the pen and apaper icon.
I don't think an image is needed here at all.

File: en_us/shared/course_components/create_problem.rst [hunk 4]
```diff
@@ -78,7 +78,7 @@ All problems on the edX platform have several component parts.
    for individual problems defaults to that number, and cannot be set to
    unlimited.
 
-   .. image:: ../../../shared/Images//AnatomyOfExercise2.png
+   .. image:: ../../../shared/images//AnatomyOfExercise2.png
```
Comment (lamagnifica) [inline en_us/shared/course_components/create_problem.rst]: search for "images//" and replace with "images/" (one slash)
you inherited this problem, and these must all be broken!!!

File: en_us/shared/course_components/create_problem.rst [hunk 5]
```diff
@@ -115,7 +115,7 @@ All problems on the edX platform have several component parts.
    is graded. If a group of problems is graded, a clock icon appears for
    that assignment in the left pane.
```
Comment (lamagnifica) [inline en_us/shared/course_components/create_problem.rst]: ^ not true, no clock icon any more
V just get rid of this image

**Comment by lamagnifica** [comment]
coupla nits, :+1:

File: en_us/olx/source/problem-xml/create_problem.rst [hunk 6]
```diff
@@ -68,15 +68,15 @@ All problems on the edX platform have several component parts.
...
+    :alt: Image of a problem from a student's point of view, with callouts for
+          attempts and showing the answer
 
 #. **Feedback.** After a learner selects **Check**, all problems return a green
    check mark or a red X.
 
-   .. image:: ../../../shared/Images/AnatomyofaProblem_Feedback.png
-    :alt: Image of feedback checkmark and x from a learner's point of view.
+   .. image:: ../../../shared/images/AnatomyofaProblem_Feedback.png
+    :alt: Image of feedback checkmark and x from a student's point of view
```
Comment (lamagnifica) [inline en_us/olx/source/problem-xml/create_problem.rst]: alt (last one)

File: en_us/shared/course_components/create_problem.rst (hunk 5 above)
Comment (catong) [inline en_us/shared/course_components/create_problem.rst]: Fine with me to get rid of the image

Comment (catong) [inline en_us/shared/course_components/create_problem.rst]: Done

**Comment by catong** [comment]
Squashed commits. Waiting to merge when tests pass

**Pull request merged**